scene/2d/box2d_fixtures.cpp

```diff
@@ -144,42 +144,35 @@ void Box2DFixture::_notification(int p_what) {
 
 				body_node = new_body;
```

Comment:
We should disconnect signals before we reassign body_node. (if body_node != null)